app/src/main/res/values/strings.xml

```diff
@@ -171,91 +171,92 @@
     <string name="chargeprice_all_tariffs_selected">all plans selected</string>
     <string name="license">License</string>
     <string name="settings_charger_data">Charging stations</string>
-    <string name="pref_data_source">Data source</string>
+    <string name="pref_data_source">Source</string>
     <plurals name="chargeprice_some_tariffs_selected">
         <item quantity="one">%d plan selected</item>
         <item quantity="other">%d plans selected</item>
     </plurals>
     <string name="unknown_operator">Unknown operator</string>
-    <string name="data_sources_description">EVMap supports multiple data sources for charging stations. Please select the one you would like to use. You can always change it later in the app\'s settings.</string>
+    <string name="data_sources_description">Please pick a source for charging stations. (Can be changed in the app settings.)</string>
     <string name="data_source_goingelectric">GoingElectric.de</string>
     <string name="data_source_openchargemap">Open Charge Map</string>
-    <string name="data_source_goingelectric_desc">Very good coverage in Germany, Austria and Switzerland and many neighboring countries. Descriptions in German. Community-maintained.</string>
-    <string name="data_source_openchargemap_desc"><![CDATA[Worldwide coverage with varying quality. Descriptions in English or local language. Community-maintained & government open data in some countries (e.g. North America, UK, France, Norway).]]></string>
+    <string name="data_source_goingelectric_desc">Great in germannic countries. Descriptions in German. Community-maintained.</string>
+    <string name="data_source_openchargemap_desc"><![CDATA[Worldwide, with varying quality. Descriptions in English or the local language. Community-maintained and open government-data in some countries (e.g. North America, UK, France, Norway).]]></string>
     <string name="next">next</string>
-    <string name="get_started">Get started</string>
-    <string name="got_it">Got it</string>
+    <string name="get_started">OK</string>
+    <string name="got_it">OK</string>
     <string name="lets_go">Let\'s go</string>
-    <string name="crash_report_text">Sorry, it seems that EVMap has crashed. Please send a crash report to the developer.</string>
-    <string name="crash_report_comment_prompt">You can add a comment below:</string>
+    <string name="crash_report_text">EVMap crashed. Please send a crash report to the developer.</string>
+    <string name="crash_report_comment_prompt">Comment below:</string>
     <string name="powered_by_mapbox">powered by Mapbox</string>
-    <string name="pref_search_provider">Place search provider</string>
-    <string name="pref_search_provider_info"><![CDATA[Data for place search, especially from Google Maps, is relatively expensive. If you use this feature often, please consider making a donation through \"About EVMap -> Donate\".]]></string>
+    <string name="pref_search_provider">Geo-searches</string>
```

Comment:
*Geo-searches* seems like a too technical term to understand for the average user (and doesn't really make it clearer)... *Location search* might be an alternative, but that could also be understood as referring to GPS data...